src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -1,12 +1,13 @@
 <tabset>
   <tab i18n-heading
-       heading="OSDs List">
+       heading="OSDs List">    
     <cd-table [data]="osds"
               (fetchData)="getOsdList()"
               [columns]="columns"
-              selectionType="single"
+              [selectionType]="'checkbox'"
```

Comment:
Please use ``multi``here. We already have datatables that support multi selection. Using checkboxes is not used in the whole Dashboard UI. See [RGW Bucket list](https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html#L6) for example.

Reply:
I have a setback in updating this PR because the multi feature doesn't seem to work. I am working on checking what's the issue. Thanks for your thorough reviews!

Reply:
After several days of scratching my head over why it wasn't working, I figured out today that pressing control button is necessary for the multi select feature , my question is whether it's so obvious for users, or if should we add some kind of a hint bubble that displays the info about the need to press and hold control button for the multi select feature?

Reply:
We should document that in our documentation. IMO check boxes would make it more obvious that the user can select multiple items, even if we are not using it anywhere yet.

@votdev the RGW Bucket list supports multiple selections? I didn't know that - That's why I'm in favor of using check boxes there as well ;)

IMO it's ok if if single selection and multi selection list differ by check boxes that would make it easier to spot if I can select multiple items.

Reply:
> my question is whether it's so obvious for users, or if we should perhaps add some kind of a hint bubble that displays the info about the need to press and hold control button for the multi select option?

Pressing 'Ctrl' button for multi selection is common workflow, you have to use it in Windows FileExplorer, Gnome Nautilus, ...
I never saw a WebUI that uses checkboxes to indicate multi selection.

Reply:
> @votdev the RGW Bucket list supports multiple selections? I didn't know that - That's why I'm in favor of using check boxes there as well ;)

You will also not see that Windows FileExplorer or Gnome Nautilus supports multi selection if you do not know it. Because of that it is no indication that a UI must show multi selection with a special control or look.

Reply:
I agree that file managers all behave somewhat equally in that degree, but there is a difference between an listing and a file manager. Sure most administrators should know that they could expect a different selection by holding down `ctrl` or `shift` and clicking something, but on the other side, they will try that once on some page. If it doesn't support multiple selections they probably wouldn't try it again on another page. 

IMHO we should notify the user somehow visually that he can do multi-selections on a particular table. It doesn't need to be check boxes, but they help selecting multiple items with a touchpad.

Reply:
I will try to add the hint using Angular's Tooltip component.